groovy/inductor_nightly.groovy

```diff
@@ -232,23 +232,23 @@ if ('op_repeats' in params) {
 }
 echo "op_repeats: $op_repeats"
 
-DT = 'float32'
-if ('DT' in params) {
-    echo "DT in params"
-    if (params.DT != '') {
-        DT = params.DT
+PRECISION = 'float32'
+if ('PRECISION' in params) {
+    echo "PRECISION in params"
+    if (params.PRECISION != '') {
+        PRECISION = params.PRECISION
     }
 }
-echo "DT: $DT"
+echo "PRECISION: $PRECISION"
 
-SHAPE = 'static'
-if ('SHAPE' in params) {
-    echo "SHAPE in params"
-    if (params.SHAPE != '') {
-        SHAPE = params.SHAPE
+TEST_SHAPE = 'static'
+if ('TEST_SHAPE' in params) {
+    echo "TEST_SHAPE in params"
+    if (params.TEST_SHAPE != '') {
+        TEST_SHAPE = params.TEST_SHAPE
     }
 }
-echo "SHAPE: $SHAPE"
+echo "TEST_SHAPE: $TEST_SHAPE"

```

Comment:
seems need update jenkins job parameters name at the same time

Reply:
I dis change the jenkins job parameters at the same time